app/services/definitions/schema_publisher.py

```diff
@@ -0,0 +1,106 @@
+from logging import Logger
+from typing import List
+
+from aries_cloudcontroller import AcaPyClient, SchemaGetResult, SchemaSendRequest
+from aries_cloudcontroller.models.txn_or_schema_send_result import TxnOrSchemaSendResult
+
+from app.exceptions import CloudApiException, handle_acapy_call
+from app.models.definitions import CreateSchema, CredentialSchema
+from app.util.definitions import credential_schema_from_acapy
+
+
+class SchemaPublisher:
+    def __init__(
+        self,
+        controller: AcaPyClient,
+        logger: Logger,
+    ):
```

Comment:
think this can also fit in 1 line. nitpick 🎨 
adding comma at end forces black to have it multiline, without it it will add the comma if it can't fit in 1 line